src/vos/vos_container.c

```diff
@@ -391,7 +391,7 @@ vos_cont_open(daos_handle_t poh, uuid_t co_uuid, daos_handle_t *coh)
 
 	rc = lrua_array_alloc(&cont->vc_dtx_array, DTX_ARRAY_LEN, DTX_ARRAY_NR,
 			      sizeof(struct vos_dtx_act_ent),
-			      LRU_FLAG_REUSE_UNIQUE,
+			      LRU_FLAG_REUSE_UNIQUE | LRU_FLAG_TRACK_ALLOC,
```

Comment:
We also need this flag in another lrua_array_alloc() instance in vos_dtx_cache_reset().

Reply:
ah, yes, thanks